src/Utility/PathUtility.php

```diff
@@ -23,10 +23,6 @@ public static function getAbsolutePath(string $path = ''): string
             return '';
         }
 
-        if (self::isUrl($path)) {
```

Comment:
We need this to allow fonts and backgrounds being used via URL

Reply:
@andi34 this destroys the purpose of the function, i need to be able to rely on getting an absolute path. not getting an absolute path here makes it useless.

Reply:
Well, then we need a different function for config defined backgrounds, placeholder and fonts to check for the url first

Reply:
will check the usages, but i don't think that's useful to have fonts, and images stored remotely that need to be processed locally - all other places that are only displaying things in the frontend do not need to have these checks. we offer to upload fonts, and images to the system through the interface, i don´t see a use case of having them stored remotely as it will slow down the system.

Reply:
even the current placement of the ftp upload drives me crazy, because the box stops working when the file can not be uploaded. normally you store the upload requests in a dedicated queue - that will be processed through a dedicated background process to avoid slowing down the system. not sure it i can manage to bring this to the first draft here to extend remote storage capabilities as i need to have this working in about 2 weeks.

Reply:
i could not spot a problem here, so i am not sure what problem is fixed with the url hack.